### ISSUE TITLE
add support for optional chaining

### ISSUE THREAD

**Issue by reslear** [nuxt-contrib/jiti#16]
hi @pi0 jiti supports creating magic in old node version?
node `v12.16.3`
npm `6.14.11`
`"nuxt-edge": "^2.15.0-26856070.8350560",`
i have a problem in express transpiling optional chaining 
```log
ERROR  ServerMiddleware Error: Unexpected token '.'   
SyntaxError: Unexpected token '.'
```
<img width="486" alt="Снимок экрана 2021-01-27 в 20 27 35" src="https://user-images.githubusercontent.com/12596485/106031750-aa3d0a80-60e0-11eb-9ab8-d9820bd12440.png">

maybe have solution for solve problem?) thx

**Comment by reslear** [comment]
maybe need functional for merge with nuxt.config build babel options?

**Comment by reslear** [comment]
@danielroe thx for answering, please show reproduction https://codesandbox.io/s/elated-rubin-uot80?file=/api/index.ts

**Comment by reslear** [comment]
@danielroe yes, need include polyfils babel or esm for example use Nullish coalescing

**Comment by pi0** [comment]
Hi @reslear. Actually this is an intended behavior since jiti only does include AST parsing with babel and minimum transformations to transform esm/typescript into into CJS module format not dealing with other language features. Adding more transpilations adds runtime overhead for newer node versions that actually support them and also we use a fast-bail method to skip transpile when cjs syntax is detected but by supporting optional chaining, we cannot do this anymore. 

So in short, i think best would be you use 14 LTS :)

**Comment by reslear** [comment]
hi @ pi0 thank you for detail answering, I thought that as a Typescript from the box will working)
ok really, probably it's time, starting migration to node 14 :)
